src/pivotal-ui/components/grids/grids.scss

```diff
@@ -2264,3 +2264,525 @@ page](/containers.html).
+  5
+ </div>
+</div>
+<div class="frow mvxl">
+ <div class="txt-r fitem-xs-6 bg-neutral-10">
+  6
+ </div>
+</div>
+<div class="frow mvxl">
+ <div class="txt-r fitem-xs-7 bg-neutral-10">
+  7
+ </div>
+</div>
+<div class="frow mvxl">
+ <div class="txt-r fitem-xs-8 bg-neutral-10">
+  8
+ </div>
+</div>
+<div class="frow mvxl">
+ <div class="txt-r fitem-xs-9 bg-neutral-10">
+  9
+ </div>
+</div>
+<div class="frow mvxl">
+ <div class="txt-r fitem-xs-10 bg-neutral-10">
+  10
+ </div>
+</div>
+<div class="frow mvxl">
+ <div class="txt-r fitem-xs-11 bg-neutral-10">
+  11
+ </div>
+</div>
+<div class="frow mvxl">
+ <div class="txt-r fitem-xs-12 bg-neutral-10">
+  12
+ </div>
+</div>
+<div class="frow mvxl">
+ <div class="txt-r fitem-xs-13 bg-neutral-10">
+  13
+ </div>
+</div>
+<div class="frow mvxl">
+ <div class="txt-r fitem-xs-14 bg-neutral-10">
+  14
+ </div>
+</div>
+<div class="frow mvxl">
+ <div class="txt-r fitem-xs-15 bg-neutral-10">
+  15
+ </div>
+</div>
+<div class="frow mvxl">
+ <div class="txt-r fitem-xs-16 bg-neutral-10">
+  16
+ </div>
+</div>
+<div class="frow mvxl">
+ <div class="txt-r fitem-xs-17 bg-neutral-10">
+  17
+ </div>
+</div>
+<div class="frow mvxl">
+ <div class="txt-r fitem-xs-18 bg-neutral-10">
+  18
+ </div>
+</div>
+<div class="frow mvxl">
+ <div class="txt-r fitem-xs-19 bg-neutral-10">
+  19
+ </div>
+</div>
+<div class="frow mvxl">
+ <div class="txt-r fitem-xs-20 bg-neutral-10">
+  20
+ </div>
+</div>
+<div class="frow mvxl">
+ <div class="txt-r fitem-xs-21 bg-neutral-10">
+  21
+ </div>
+</div>
+<div class="frow mvxl">
+ <div class="txt-r fitem-xs-22 bg-neutral-10">
+  22
+ </div>
+</div>
+<div class="frow mvxl">
+ <div class="txt-r fitem-xs-23 bg-neutral-10">
+  23
+ </div>
+</div>
+<div class="frow mvxl">
+ <div class="txt-r fitem-xs-24 bg-neutral-10">
+  24
+ </div>
+</div>
+```
+
+Row-major:
+
+```html_example
+<div class="fcol">
+ <div class="fitem-xs frow mvxl">
+  <div class="fitem-xs"><div style="height: 300px" class="pal bg-neutral-10">Tall</div></div>
+  <div class="fitem-xs"><div style="height: 200px" class="pal bg-neutral-10">World</div></div>
+ </div>
+ <div class="fitem-xs frow mvxl">
+  <div class="fitem-xs"><div style="height: 300px" class="pal bg-neutral-10">Tall</div></div>
+  <div class="fitem-xs"><div style="height: 200px" class="pal bg-neutral-10">World</div></div>
+ </div>
+</div>
+```
+
+Wrapping and offsetting:
+
+```html_example
+<div class="frow">
+ <div class="fitem-xs-offset-6 fitem-xs-12 mbxl">
+  <div class="bg-neutral-10">Offset 6, Size 12</div>
+ </div>
+ <div class="fitem-xs-6 mbxl">
+  <div class="bg-neutral-10">Offset 0, Size 6</div>
+ </div>
+ <div class="fitem-xs-4 mbxl">
+  <div class="bg-neutral-10">Offset 0, Size 4</div>
+ </div>
+ <div class="fitem-xs-offset-4 fitem-xs-16 mbxl">
+  <div class="bg-neutral-10">Offset 4, Size 16</div>
+ </div>
+</div>
+```
+
+Alignment and justification:
+
+```html_example
+<div class="frow flex-justify-start-xs">
```

Comment:
we need words describing these options... it will provide the clarity along with the rendered examples... e.g. as a user, I see the rendered example, I think I understand what it does, then I read the description and voila, verify that I've understood. Or, likely in your case the other way around. ;)